.github/workflows/unit-tests-workflow.yml

```diff
@@ -2,19 +2,25 @@ name: Unit tests workflow
 on:
   push:
     branches:
-      - main
-      - 1.x
+      - "*"
   pull_request:
     branches:
-      - main
-      - 1.x
+      - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '1.x'
+  OPENSEARCH_DASHBOARDS_VERSION: '1.3'
 jobs:
   tests:
     name: Run unit tests
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
```

Comment:
should we not have `macos-latest` since they always fail?

Reply:
Was just thinking the same thing. Removing it now.